### PR TITLE
Update old-simulator-doc.txt

### PR DESCRIPTION
**Pull request by Acquiescence3**
I changed Pokemon to Pokémon, as requested.

### CODE REVIEW

File: old-simulator-doc.txt [hunk 1]
```diff
@@ -68,21 +68,21 @@ foeModifyMove(user, target, move) [on target]
 
 	examples: [ability] Adaptability, [ability, foe ability] No Guard
 
-modifyPokemon(pokemon) [on pokemon]
-	Any effect that modifies a pokemon should be applied here.
+modifyPokémon(Pokémon) [on Pokémon]
```
Comment (TheJetOU) [inline old-simulator-doc.txt]: Function names probably shouldn't be changed.

Comment (Acquiescence3) [reply old-simulator-doc.txt]: Oops sorry. I will fix that in a bit

Comment (Acquiescence3) [reply old-simulator-doc.txt]: I fixed the function names, but accidentally deleted some of the code on lines 71 and 72. I will see if I can retrieve it.

**Comment by Slayer95** [comment]
Not blocked on #5439 AFAIAC (also re: that PR, damn procrastination!)

File: old-simulator-doc.txt [hunk 2]
```diff
@@ -68,21 +68,21 @@ foeModifyMove(user, target, move) [on target]
...
-	modifyPokemon is fired.
+	Whenever any effect is added or removed, the Pokémon is reset and
+	modifypokemon is fired.
 
 	examples: [ability] Light Metal, [volatile] Conversion 2, etc
 
-stats(stats, pokemon) [on pokemon]
-	A shortcut event for modifyPokemon(pokemon) : stats = pokemon.stats.
+stats(stats, pokemon) [on Pokémon]
+	A shortcut event for modifypokemon(pokemon) : stats = pokemon.stats.
```
Comment (Zarel) [inline old-simulator-doc.txt]: This is wrong though? What were you trying to do here?

**Comment by Zarel** [comment]
I also don't know if I agree with this in general.

I feel like, in internal documentation, it's more readable to use "Pokemon" for the class, and "pokemon" for instances of that class. We can reserve "Pokémon" for user-facing documentation.

Out of curiosity, who requested this?

**Comment by scheibo** [comment]
> What exactly are you trying to do, Kirkburn?

_Originally posted by @Marty-D in https://github.com/Zarel/Pokemon-Showdown/pull/5565#issuecomment-507408471_

**Comment by Zarel** [comment]
scheibo, HoeenHero, bumba, and I, representing a plurality of programming staff, agree that `pokemon` is better than `Pokémon` in non-user-facing docs.

I'm assuming "as requested" is a social engineering attempt, which, nice try. But if someone did ask you, I'm still interested in who.

**Comment by Acquiescence3** [comment]
Nobody asked me specifically, I just saw that it needed to have the word Pokemon changed.

File: old-simulator-doc.txt (hunk 2 above)
Comment (Acquiescence3) [reply old-simulator-doc.txt]: Whoops I meant to change the "Pokémon" in the function commands back to "pokemon". I must have gotten a little confused.

Sorry about that, I would like to help fix some bugs here but am still new to coding.

**Pull request closed without merging**